src/main/java/com/aws/greengrass/status/FleetStatusService.java

```diff
@@ -705,22 +724,24 @@ private DeploymentInformation getDeploymentInformation(Map<String, Object> deplo
                 .deploymentId((String) deploymentDetails.get(GG_DEPLOYMENT_ID_KEY_NAME))
                 .fleetConfigurationArnForStatus((String) deploymentDetails.get(CONFIGURATION_ARN_KEY_NAME)).build();
         if (deploymentDetails.containsKey(DEPLOYMENT_STATUS_DETAILS_KEY_NAME)) {
-            Map<String, Object> statusDetailsMap =
-                    (Map<String, Object>) deploymentDetails.get(DEPLOYMENT_STATUS_DETAILS_KEY_NAME);
+            Map<String, Object> statusDetailsMap = (Map<String, Object>) deploymentDetails
+                    .get(DEPLOYMENT_STATUS_DETAILS_KEY_NAME);
             StatusDetails statusDetails = StatusDetails.builder()
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Potential log injection detected. Ensure all untrusted input is properly sanitized before logging. Use parameterized logging or validate input against a allow list to prevent log injection vulnerabilities. Consider using a dedicated logging library&#39;s built-in sanitization features when available. Learn more - https://cwe.mitre.org/data/definitions/117.html